packages/console/src/csv/CsvOverlay.tsx

```diff
@@ -1,19 +1,45 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import React, {
+  ChangeEvent,
+  Component,
+  DragEvent,
+  MouseEvent,
+  ReactElement,
+  RefObject,
+} from 'react';
 import memoize from 'memoize-one';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { ContextActions, GLOBAL_SHORTCUTS } from '@deephaven/components';
+import {
+  ContextAction,
+  ContextActions,
+  GLOBAL_SHORTCUTS,
+} from '@deephaven/components';
 import {
   dhFileCsv,
   dhFileDownload,
   dhFileSpreadsheet,
+  IconDefinition,
   vsClippy,
   vsFileZip,
   vsTrash,
   vsWarning,
 } from '@deephaven/icons';
 import './CsvOverlay.scss';
-import { TextUtils } from '@deephaven/utils';
+import { assertNotNull, TextUtils } from '@deephaven/utils';
+
+interface CsvOverlayProps {
+  allowZip: boolean;
+  onFileOpened: (file: File | null) => void;
```

Comment:
As mentioned in another comment, add a new prop `onCancel` that is used instead of `onFileOpened(null)`